rhel8/kickstart/ssg-rhel8-anssi_bp28_enhanced-ks.cfg

```diff
@@ -6,9 +6,6 @@
 # https://pykickstart.readthedocs.io/en/latest/
 # http://usgcb.nist.gov/usgcb/content/configuration/workstation-ks.cfg
 
-# Install a fresh new system (optional)
```

Comment:
What is the reaspon of removing this part? It stays in rhel7 kickstarts.

Reply:
Sorry, I should have mentioned that.
The `install` command is deprecated in RHEL8.
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_installation/kickstart-commands-and-options-reference_installing-rhel-as-an-experienced-user#install_kickstart-commands-for-installation-program-configuration-and-flow-control

Reply:
Having it doesn't cause any harm, but just a warning message.

Reply:
Thank you for explanation.